### PR TITLE
desktop: fix big input, and other desktop web issues

### PR DESCRIPTION
**Pull request by patosullivan**
fixes TLON-3273 (making the BigInput actually usable on desktop) We weren't accounting for big input height.

Also fixes other issues:

Notebooks weren't wrapped in a Pressable. 

Some bad conditional checks were causing text strings to render outside of Text components/directly within Stack components.

Edits weren't working at all for notebook posts (we weren't filling in the draft content, we weren't passing along the title, and we weren't handling situations where the post had an image).

We were still unnecessarily using a ScrollView for the web version of the BigInput.

We had no way to allow the user to exit the big input editor if they wanted to abandon creating a post.

We were attempting to pass `contentFit` as a prop to `img` tags for web, which isn't a valid attribute, which made the image header in the notebook editor too large. I added a way to catch `contentFit` and pass it to the `img` as an `objectFit` property in the style prop. This also fixes an issue where image avatars weren't displaying correctly much of the time.

We had no overflow menu button on notebook posts, so there was no way to get to the message options on desktop.

General cleanup of the web MessageInput component.

Edit: now also adds an overflow button for gallery posts.

### CODE REVIEW

**Review by latter-bolden — APPROVED** [review]
Looks good, didn't see any regressions checking against native.

Only UX thing that threw me off was clicking into an empty big input didn't focus it unless I hit within the height of the first line.

**Review by dnbrwstr — COMMENTED** [review]
Couple comments

File: packages/ui/src/components/NotebookPost/NotebookPost.tsx [hunk 1]
```diff
@@ -89,69 +95,77 @@ export function NotebookPost({
...
+        <SendPostRetrySheet
+          open={showRetrySheet}
+          post={post}
+          onOpenChange={setShowRetrySheet}
+          onPressRetry={handleRetryPressed}
+          onPressDelete={handleDeletePressed}
+        />
+      </Pressable>
+      {isWeb && !hideOverflowMenu && (
+        <View position="absolute" top={8} right={24} width={8} height={0}>
```
[the remark this answers is not in the document]
Comment (patosullivan) [reply packages/ui/src/components/NotebookPost/NotebookPost.tsx]: yep

File: packages/ui/src/components/GalleryPost/GalleryPost.tsx [hunk 2]
```diff
@@ -77,48 +80,62 @@ export function GalleryPost({
...
+            open={showRetrySheet}
+            onOpenChange={setShowRetrySheet}
+            post={post}
+            onPressDelete={handleDeletePressed}
+            onPressRetry={handleRetryPressed}
+          />
+        </GalleryPostFrame>
+      </Pressable>
+      {isWeb && !hideOverflowMenu && (
+        <View position="absolute" top={8} right={24} width={8} height={0}>
```
[the remark this answers is not in the document]
Comment (patosullivan) [reply packages/ui/src/components/GalleryPost/GalleryPost.tsx]: ended up disabling the onPress handler for the pressable while the user is hovering over the OverflowMenuButton

**Pull request merged**